src/components/SwapWidget/SwapWidget.tsx

```diff
@@ -710,6 +669,83 @@ const SwapWidget: FC = () => {
     }
   };
 
+  useEffect(() => {
```

Comment:
What's the actual change here?

Reply:
I'm not sure about this useEffect hook. I never worked on it. I moved some code around, so my name might appear next to it on Git